examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -43,3 +46,19 @@
 
 # Now let's take a look at the spatial distributions of the psd.
 epochs.plot_psd_topomap(ch_type='grad', normalize=True)
+
+# Alternatively, you may also create PSDs from Epochs objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(epochs[:5], low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
`picks` are for selecting channels and the `:5` slicing is for which epochs to use, do they have some dependence that you are thinking of?